bofire/strategies/doe/objective.py

```diff
@@ -83,6 +74,71 @@ def evaluate_jacobian(self, x: np.ndarray) -> np.ndarray:
     def _evaluate_jacobian(self, x: np.ndarray) -> np.ndarray:
         pass
 
+    @abstractmethod
```

Comment:
Are these functions needed in the Objective class? Since the SpaceFilling objective actually does not have to implement any of these functions, right? (The only function that is used for any objective is get_model_matrix() in the exit message find_local_max_ipopt() i think, but imo it is not necessary there as well...). What is your opinion?